docs/visual-basic/developing-apps/programming/app-settings/accessing-application-settings.md

```diff
@@ -52,9 +52,6 @@ This section contains topics describing the `My.Settings` object and the tasks i
 |Save updated user setting values|[How to: Persist User Settings in Visual Basic](../../../../visual-basic/developing-apps/programming/app-settings/how-to-persist-user-settings.md)|  
 |Determine the values of user settings|[How to: Read Application Settings in Visual Basic](../../../../visual-basic/developing-apps/programming/app-settings/how-to-read-application-settings.md)|  
   
-## See Also  
- [Managing Application Settings (.NET)](/visualstudio/ide/managing-application-settings-dotnet)   
```

Comment:
Sounds good. I've been planning to send an email to the team on the cross-repo links. Check your inbox and send me your thoughts!